airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -1808,6 +1808,7 @@ def __init__(
         gcp_conn_id: str = "google_cloud_default",
         impersonation_chain: Optional[Union[str, Sequence[str]]] = None,
         asynchronous: bool = False,
+        cancel_on_kill: bool = True,
```

Comment:
Can you add please this parameter to docstring so user can understand what it does?